docs/hub/datasets-cards.md

```diff
@@ -1,7 +1,30 @@
 # Dataset Cards
 
-Each dataset may be documented by the `README.md` file in the repository. This file is called a **dataset card**, and the Hugging Face Hub will render its contents on the dataset's main page. To inform users about how to responsibly use the data, it's a good idea to include information about any potential biases within the dataset. Generally, dataset cards help users understand the contents of the dataset and give context for how the dataset should be used.
+Each dataset may be documented by the `README.md` file in the repository. This file is called a **dataset card**, and the Hugging Face Hub will render its contents on the dataset's main page. To inform users about how to responsibly use the data, it's a good idea to include information about any potential biases within the dataset. Generally, dataset cards help users understand the contents of the dataset and give context for how the dataset should be used. 
 
-The [Dataset Card Creation Guide](https://github.com/huggingface/datasets/blob/main/templates/README_guide.md) gives an overview of what components are found in a dataset card. The process for creating a new dataset card is outlined in the [Create a dataset card](https://huggingface.co/docs/datasets/dataset_card) guide.
+You can also add dataset tags to your card. Dataset tags describe important information about a dataset such as its license, language, and size. More importantly, the tags help users discover a dataset on the Hub. Tags are defined in a YAML section at the top of the `README.md` file. Supported tags include:
+
+```yaml
+annotations_creators: List[str]
+language_creators: List[str]
+language: List[str]
```

Comment:
```suggestion
language: List[str]
language_details: Optional[str]
```

Reply:
I think some reference to language_detials would be useful